test/e2e/__init__.py

```diff
@@ -38,6 +39,17 @@ def sagemaker_client():
     return boto3.client("sagemaker")
 
 
+def cross_region_sagemaker_client():
+    return boto3.client("sagemaker", region_name=get_cross_region())
+
+
+def get_cross_region():
```

Comment:
+1
I think this method is better suited to be in the test file itself

Reply:
Sounds good, If there is any reason for using cross_region in other tests in the future we can move it back here again